### DISCUSSION TITLE
在umi3中使用antd4，引用任何组件会报TS警告

### DISCUSSION THREAD

**Discussion by greenhand-xj** [ant-design/ant-design#43160] [Q&A]
![image](https://github.com/ant-design/ant-design/assets/81698900/9dc722c7-b8c1-4e35-9ed3-6626ad1f1963)
![image](https://github.com/ant-design/ant-design/assets/81698900/26cf477f-7dff-4156-b6df-af8c306598a8)

**Comment by zombieJ** [comment]
感觉是依赖问题，重新安装 `node_modules` & 升级依赖一般可解。

**Reply by greenhand-xj** [reply]
好的，我试试

**Reply by greenhand-xj** [reply]
感谢回复

**Comment by kekezjqqqqq** [comment]
在tsconfig.json中的"paths"加上 "react": [
"./node_modules/@types/react"
]可以解决

![image](https://github.com/ant-design/ant-design/assets/137025257/526422e2-d92c-4406-bbe6-c8735363172d)

**Reply by greenhand-xj** [reply]
好的，我试试

**Reply by greenhand-xj** [reply]
这样是可以的，感谢

**Reply by greenhand-xj** [reply]
但这是为什么会这样啊

**Reply by greenhand-xj** [reply]
我没有动任何东西，就是用Umi-cli初始化的
